libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/factory/RouteProgressInstanceFactory.kt

```diff
@@ -0,0 +1,75 @@
+package com.mapbox.navigation.base.internal.factory
+
+import com.mapbox.api.directions.v5.models.BannerInstructions
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.VoiceInstructions
+import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.trip.model.RouteLegProgress
+import com.mapbox.navigation.base.trip.model.RouteProgress
+import com.mapbox.navigation.base.trip.model.RouteProgressState
+import com.mapbox.navigation.base.trip.model.roadobject.UpcomingRoadObject
+
+/**
+ * Internal factory to build [RouteProgress] objects
+ */
+object RouteProgressInstanceFactory {
```

Comment:
Same as above.

Reply:
```suggestion
object RouteProgressFactory {
```
To keep the naming the same as other factories.

Reply:
There were other factories with 'instance' in the name like `EHorizonInstanceFactory` and `RoadObjectInstanceFactory`. I removed 'instance' from all of them.